lib/topological_inventory.rb

```diff
@@ -14,8 +14,13 @@ def self.call
 
   private_class_method def self.raw_api
     TopologicalInventoryApiClient.configure do |config|
-      config.host   = ENV['TOPOLOGICAL_INVENTORY_URL'] || 'localhost'
+      config.host = ENV['TOPOLOGICAL_INVENTORY_URL'] || 'localhost'
       config.scheme = URI.parse(ENV['TOPOLOGICAL_INVENTORY_URL']).try(:scheme) || 'http'
+      # Set up user/pass for basic auth if we're in dev and they exist.
+      if Rails.env.development?
+        config.username = ENV['DEV_TOPOLOGICAL_INVENTORY_USERNAME'] || nil
```

Comment:
@syncrou @gmcculloug @Fryguy @lindgrenj6 
I researched this with using JWT tokens that we can get from Insights proxy but that token expires quickly.
After discussing with @Fryguy the option that we have is to environment variables only for Dev environment where we would need the service portal to connect to Topology Service we would repeat this pattern for other services like RBAC service or Approval Service